.vscode/launch.json

```diff
@@ -861,14 +857,12 @@
       "name": "Tests: mds-repository",
       "type": "node",
       "request": "launch",
-      "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
-      "args": [
-        "--project",
-        "${workspaceFolder}/tsconfig.json",
-        "-r",
-        "ts-node/register",
-        "--timeout",
-        "0"
+      "runtimeArgs": [
+        "--inspect-brk",
+        "${workspaceRoot}/node_modules/jest/bin/jest",
+        "--runInBand",
+        "--coverage",
+        "false"
```

Comment:
Why no coverage here? Would it be too low to be worth turning it on / setting a value in jest config?

Reply:
Is it because it's for VSCode? IN that case ignore these comments.

Reply:
These are VS Code debugger launch configurations. Probably not interested in coverage here. Coverage still performed for running tests outside of the debugger.